mimesis/data/en/text.json

```diff
@@ -1,67 +1,63 @@
 {
-  "alphabet": {
-	"uppercase": [
-	  "A",
-	  "B",
-	  "C",
-	  "D",
-	  "E",
-	  "F",
-	  "G",
-	  "H",
-	  "I",
-	  "J",
-	  "K",
-	  "L",
-	  "M",
-	  "N",
-	  "O",
-	  "P",
-	  "Q",
-	  "R",
-	  "S",
-	  "T",
-	  "U",
-	  "V",
-	  "W",
-	  "X",
-	  "Y",
-	  "Z"
+	"alphabet": {
+	  "uppercase": [
```

Comment:
Why do you replace English alphabet?

Reply:
Oopsies, must have confused the files. I reset them